cmd/serverless/main.go

```diff
@@ -297,7 +297,13 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 
 	// start the proxy if needed
 	// as this feature is experimental, this should be the only entrypoint (easier to remove)
-	_ = proxy.Start("127.0.0.1:9000", "127.0.0.1:9001", &invocationlifecycle.ProxyProcessor{})
+	_ = proxy.Start(
+		"127.0.0.1:9000",
+		"127.0.0.1:9001",
+		&invocationlifecycle.ProxyProcessor{
+			Tags:           serverlessDaemon.ExtraTags.Tags,
+			MetricsChannel: serverlessDaemon.MetricAgent.GetMetricChannel(),
```

Comment:
Maybe a nil check against  `serverlessDaemon.MetricAgent` would be nice before accessing the method?